app/views/practices/show/introduction/_introduction.html.erb

```diff
@@ -83,6 +83,11 @@
               <% else %>
                 <%= origin_display_name(@practice) %>
               <% end %>
+            <!-- display department --->
+            <% dept_name = origin_display_department_name(@practice) %>
+            <% if dept_name.length() > 0 %>
```

Comment:
remove `()`

Reply:
Can we do

```
<!-- display department --->
<% if @practice.department? %>
    <span>, <%= origin_display_department_name(@practice) %> Department</span>
<% end %>
```
?

Reply:
Yes to both - I will make those fixes ^^